helios-client/src/main/java/com/spotify/helios/common/descriptors/RolloutOptions.java

```diff
@@ -31,20 +31,24 @@
 
   public static final long DEFAULT_TIMEOUT = TimeUnit.MINUTES.toSeconds(5);
   public static final int DEFAULT_PARALLELISM = 1;
+  public static final int DEFAULT_FAILURE_THRESHOLD = 1;
 
   private final long timeout;
   private final int parallelism;
   private final boolean migrate;
   private final boolean overlap;
+  private final int failureThreshold;
 
   public RolloutOptions(@JsonProperty("timeout") final long timeout,
                         @JsonProperty("parallelism") final int parallelism,
                         @JsonProperty("migrate") final boolean migrate,
-                        @JsonProperty("overlap") boolean overlap) {
+                        @JsonProperty("overlap") boolean overlap,
+                        @JsonProperty("failureThreshold") final int failureThreshold) {
```

Comment:
Ah thanks. We use @Nullable and `    this.failureThreshold = Optional.fromNullable(failureThreshold).or(0);` because `@DefaultValue()` only takes String.